### ISSUE TITLE
PC cold boots during MNIST example on GPU (GF980Ti); PSU problem; reduce load?

### ISSUE THREAD

**Issue by SeveQ** [tensorflow/tensorflow#740]
After getting Tensorflow to work at all, now I've got another issue: I can run through the 20.000 training steps (within about 1 second; holy sh... that thing's fast) of the Deep MNIST example (code below). But after the last step my PC does a cold reboot. It just turns off hard without shutdown and reboots after a few seconds.

It appears to be caused by an undersized PSU. Gaming on the GPU however works fine so far... 

So, is there any way that I can reduce the load on the GPU during usage of Tensorflow? I'd be fine if the MNIST example took 2 seconds instead of 1 if only my PC wouldn't reboot. 

At the moment unfortunately I cannot afford a new PSU, spent my whole available budget on the GF980Ti. 

Any ideas?

**Comment by SeveQ** [comment]
Oops, forgot the code. Here you go, the code I use to test my Tensorflow setup: http://pastebin.com/Wad9eRBE

It's not my own code. I'm not sure where I got it but at first glance it looked pretty much like the Deep MNIST example from the tutorials. Strange thing is: it converges to an accuracy of about 0.9, whereas the actual Deep MNIST example from the tutorials converges to about 0.99. 

Anyway... I was, again, able to kind of solve it myself by throttling the GPU maximum power using the command 

    ~$ nvidia-smi -pl 220

which, if I got it right, limits the power consumption of the GPU to 220 watts.

**Comment by stmharry** [comment]
@HWiese1980 
I have been encountering reboot problems like yours. However, reboot happens sometimes as early as importing tensorflow (loading *.so), and sometimes after running a few iterations of training. In this case do you this it's a PSU problem like yours? Since I have tested out loading Theano and the same problem occurs on loading Theano.

**Comment by HWiese1980** [comment]
@stmharry Yes, I'd still say it's a PSU problem. The evidence is pretty much the same. Definitely sounds like an undersized PSU. I bought a Corsair HX750i several weeks ago and since then never again experienced these sudden reboots. So I highly recommend upgrading your PSU to something around at least 700 watts.
